index.bs

```diff
@@ -4039,6 +4050,11 @@ When the [=create a new browsing context=] algorithm is invoked, after the
 
 1. Let |context| be the newly created browsing context.
 
+1. Set |context|'s [=original opener=] to |context|'s [=opener browsing
+   context=], as if the |context|'s [=opener browsing context=] was
+   created according to [=the rules for choosing a navigable=] but with
+   <code>noopener</code> always set to false.
```

Comment:
@jgraham WDYT about defining the original opener like this? Another way to specify it would be to update the HTML-spec but it would require many changes since the opener is discarded before the algorithm descends to notify WebDriver BiDi about the browsing context creation.